lib/src/embedded/compiler.test.ts

```diff
@@ -2,50 +2,73 @@
 // MIT-style license that can be found in the LICENSE file or at
```

Comment:
It looks like the tests in this file aren't actually making any assertions. What are you trying to validate with them?

Reply:
This is a smoke test for invoking the child process, to help assess whether the Node process itself is starting up or not. I figured it would help debug platform [Windows] issues if they come up, or if there's an issue with the embedded binary.

Reply:
It's still a good practice to make some sort of assertion that the output (whether it's stdout or whatever) looks how you'd expect it to.